pkg/monitor/api.go

```diff
@@ -24,10 +24,19 @@ import (
 	configclientset "github.com/openshift/client-go/config/clientset/versioned"
 )
 
+const (
+	LocatorKubeAPIServerNewConnection         = "kube-apiserver-new-connection"
+	LocatorOpenshiftAPIServerNewConnection    = "openshift-apiserver-new-connection"
+	LocatorOAuthAPIServerNewConnection        = "oauth-apiserver-new-connection"
+	LocatorKubeAPIServerReusedConnection      = "kube-apiserver-reused-connection"
+	LocatorOpenshiftAPIServerReusedConnection = "openshift-apiserver-reused-connection"
+	LocatorOAuthAPIServerReusedConnection     = "oauth-apiserver-reused-connection"
+)
+
 // Start begins monitoring the cluster referenced by the default kube configuration until
 // context is finished.
 func Start(ctx context.Context) (*Monitor, error) {
-	m := NewMonitor()
+	m := NewMonitorWithInterval(time.Second)
```

Comment:
maybe this is somehow not causing the "normal" flakes we see in the e2e monitors in master?